rdr_service/dao/genomics_dao.py

```diff
@@ -1446,3 +1446,15 @@ def get_from_filepath(self, filepath):
             ).filter(
                 GenomicAW2Raw.file_path == filepath
             ).all()
+
+
+class GenomicIncidentDao(UpdatableDao):
+    def __init__(self):
+        super(GenomicIncidentDao, self).__init__(
+            GenomicIncident, order_by_ending=['id'])
+
+    def get_id(self, obj):
+        pass
+
+    def from_client_json(self):
+        pass
```

Comment:
Is this to mitigate the `NotImplementedError` on the parent class?

Reply:
Yes, exactly.